_posts/altair-to-mpl-scatter-part2.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "Making a Scatter Plot - Part 2"
+date:   2018-08-15 17:00:00 -0500
+author: "Kimberly Orr and Nabarun Pal"
+categories: user-guide
+tags: "intro about scatter"
+excerpt_separator: <!--read more-->
+---
+
+# Making a Complex Scatter Plot
+At the time of writing, mpl-altair does not support scatter plots with nominal or ordinal color encodings, so this post will show how to create a complex scatter plot in Altair, Matplotlib, and how mpl-altair _should_ implement the chart conversion in the future.
+
+In the first part, we made a simple scatter plot. This post will look at a more complex plot.
+We'll use the cars dataset again:
+```python
+from vega_datasets import data
+cars = data.cars()
+cars.head()
+```
+**Acceleration** | **Cylinders** | **Displacement** | **Horsepower** | **Miles_per_Gallon** | **Name** | **Origin** | **Weight_in_lbs** | **Year** 
+:---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---:
+12.0 | 8 | 307.0 | 130.0 | 18.0 | chevrolet chevelle malibu | USA | 3504 | 1970-01-01
+11.5 | 8 | 350.0 | 165.0 | 15.0 | buick skylark 320 | USA | 3693 | 1970-01-01
+11.0 | 8 | 318.0 | 150.0 | 18.0 | plymouth satellite | USA | 3436 | 1970-01-01
+12.0 | 8 | 304.0 | 150.0 | 16.0 | amc rebel sst | USA | 3433 | 1970-01-01
+10.5 | 8 | 302.0 | 140.0 | 17.0 | ford torino | USA | 3449 | 1970-01-01
+
+In addition to looking at Horsepower vs Weight, let's color each point by its origin country.
+
+## Altair
+Since Altair is based on linking columns to encodings, we just have to specify 
+that the color encoding comes from the Origin column.
+
+Also, notice that a legend is automatically generated.
+```python
+import altair as alt
+alt.Chart(cars).mark_point().encode(
+    alt.X('Weight_in_lbs'),
+    alt.Y('Horsepower'),
+    alt.Color('Origin')
+)
+```
+![png](pics/altair-to-mpl-scatter-part2_0.png)
+
+## Matplotlib
+If we wanted to color our points by a quantitative variable, we could have just 
+used 
+```python
+ax.scatter('Weight_in_lbs', 'Horsepower', c='quantitative_column', data=cars)
+```
+However, the scatter function currently doesn't allow scalar mappables for categorical data. 
```

Comment:
Categorical color definitions (my guess is that scalar mappables is too weedy for the audience for this post)